docs/build.md

```diff
@@ -206,6 +206,14 @@ This provides GPU acceleration using the MUSA cores of your Moore Threads MTT GP
   cmake --build build --config Release
   ```
 
+  For static build:
```

Comment:
You can follow: https://github.com/BodhiHu/llama.cpp/blob/musa/docs/build.md?plain=1#L21-L46 to update the doc.

Reply:
Hi @yeahdongcn , as we aligned, there's not a unified vendor build doc spec, I'm not sure how this should be updated further.